src/helm/env.d/dev/secrets.enc.yaml

```diff
@@ -1,7 +1,8 @@
-djangoSecretKey: ENC[AES256_GCM,data:06KBEHV/gBgGoB4DXf9yTU5XK1xP9OXfyKEiSdSghV8XIMon3o1ajSWN+WNMRHkRZuU=,iv:ZeP1X4pQF9fVm7quzzVXSm2CSLrqAizwZD5QFmNOoSc=,tag:Dm/b+6CfznSC+CdKj1SCYA==,type:str]
+djangoSecretKey: ENC[AES256_GCM,data:9fOtt8oesY2CUahg972UGldDrqqF6Fa1Tn+bKxNpMbfXppQtPY2Jfu4EWKAaqH07X00=,iv:OC0ggDgCcja6h4IK73jVXZGDE1qp5OJfeNg182DKxQ4=,tag:ITMAWmPxW8lNBvm2Xefw/Q==,type:str]
+djangoSuperUserPass: ENC[AES256_GCM,data:mkLVMnc=,iv:qYBUdUwJk422RVm23/6CUKubFtBL+lofynSnkJglNQk=,tag:Md5FPXwCe9kl5BkICHszzg==,type:str]
```

Comment:
I have a quick question: Could you clarify why we're adding the `djangoSuperUserPass` secret to the development values in this commit? I'm trying to understand its connection.

Additionally, how do we ensure that secrets remain hidden during deployments? Might be great to add it in commits body for our future ourselves

Does creating the secrets file mean we need to update the development `secrets.enc` values? And how does this impact the staging environment?